From f5bb73e1f8a4008bee385f592004526ab92b49ad Mon Sep 17 00:00:00 2001
From: "Simental Magana, Marcos" <marcos.simental.magana@intel.com>
Date: Tue, 18 Jun 2019 09:45:19 -0500
Subject: [PATCH 2/2] use clearlinux's ld.so.cache

/etc/ld.so.cache tends to not exists (stateless).

Probably a better fix would be to always use
/var/cache/ldconfig/ld.so.cache and append /etc/ld.so.cache file if
it exists. In the meantime we use ours.

Signed-off-by: Simental Magana, Marcos <marcos.simental.magana@intel.com>
---
 src/gnome-desktop/gnome-desktop-thumbnail-script.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gnome-desktop/gnome-desktop-thumbnail-script.c b/src/gnome-desktop/gnome-desktop-thumbnail-script.c
index 8e8b876..7dd3eab 100644
--- a/src/gnome-desktop/gnome-desktop-thumbnail-script.c
+++ b/src/gnome-desktop/gnome-desktop-thumbnail-script.c
@@ -538,7 +538,7 @@ add_bwrap (GPtrArray   *array,
   add_args (array,
 	    "bwrap",
 	    "--ro-bind", "/usr", "/usr",
-	    "--ro-bind", "/etc/ld.so.cache", "/etc/ld.so.cache",
+	    "--ro-bind", "/var/cache/ldconfig/ld.so.cache", "/var/cache/ldconfig/ld.so.cache",
 	    NULL);
 
   /* These directories might be symlinks into /usr/... */
-- 
2.22.0

